### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by kshred**
Trying to add my info to the attendance list.

### CODE REVIEW

File: workshop_attendance.txt [hunk 1]
```diff
@@ -1,4 +0,0 @@
-# Please enter your name and your github handle, each in a new line.
-
-Blanca Mancilla
-@bluciam
```
Comment (francoiseprovencher) [inline workshop_attendance.txt:4]: Oh no! You deleted the original file :-( Is it possible to revert the deletion of `workshop_attendance.txt`?

Comment (kshred) [inline workshop_attendance.txt:4]: Oh...Sorry.  I think I can revert it.

2015-10-08 20:37 GMT-04:00 Françoise Provencher notifications@github.com:

> In workshop_attendance.txt
> https://github.com/bluciam/helloworld_pyladiesMTL/pull/39#discussion_r41587214
> :
> 
> > @@ -1,4 +0,0 @@
> > 
> > ## -# Please enter your name and your github handle, each in a new line.
> > 
> > -Blanca Mancilla
> > -@bluciam
> 
> Oh no! You deleted the original file :-( Is it possible to revert the
> deletion of workshop_attendance.txt?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/bluciam/helloworld_pyladiesMTL/pull/39/files#r41587214
> .

## 

molt.bandcamp.com

Comment (bluciam) [inline workshop_attendance.txt:4]: Should be delete this pull requests?

Comment (kshred) [inline workshop_attendance.txt:4]: Yes, please delete that pull request.  You should have received a second
pull request from me with the file kshred_attendance.txt (or something to
that effect.)

Thanks,
Kyla

2015-10-09 12:20 GMT-04:00 Blanca Mancilla notifications@github.com:

> In workshop_attendance.txt
> https://github.com/bluciam/helloworld_pyladiesMTL/pull/39#discussion_r41648910
> :
> 
> > @@ -1,4 +0,0 @@
> > 
> > ## -# Please enter your name and your github handle, each in a new line.
> > 
> > -Blanca Mancilla
> > -@bluciam
> 
> Should be delete this pull requests?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/bluciam/helloworld_pyladiesMTL/pull/39/files#r41648910
> .

## 

molt.bandcamp.com

**Comment by bluciam** [comment]
OK @kshred , closing this one!

**Pull request closed without merging**